Documentation/Architecture.md

```diff
@@ -0,0 +1,15 @@
+# [Zowe](https://www.zowe.org/) Explorer Extension Templates Architecture
+
+![alt text](Architecture.png)

```

Comment:
On the architecture drawing, you can remove the box for `cURL`:
![image](https://user-images.githubusercontent.com/14940561/126217282-3eaffed2-f5e4-4f5f-8a32-73b4a9219e83.png)

Zowe CLI + any plugins for it are an alternative to cURL in that Zowe CLI calls a REST API, but has specific logic for mainframe opreations.